src/main/java/org/jenkinsci/plugins/workflow/log/ConsoleAnnotators.java

```diff
@@ -98,7 +100,7 @@ public static void setAnnotator(ConsoleAnnotator<?> annotator) throws IOExceptio
         }
         StaplerResponse rsp = Stapler.getCurrentResponse();
         if (rsp != null) {
-            rsp.setHeader("X-ConsoleAnnotator", new String(Base64.encode(baos.toByteArray())));
+            rsp.setHeader("X-ConsoleAnnotator", new String(Base64.getEncoder().encode(baos.toByteArray()), Charset.defaultCharset()));
```

Comment:
Never use the default charset for anything on the master.

```suggestion
            rsp.setHeader("X-ConsoleAnnotator", new String(Base64.getEncoder().encode(baos.toByteArray()), StandardCharsets.US_ASCII));
```